src/utils/zendesk.ts

```diff
@@ -0,0 +1,13 @@
+import { loadScript } from "./loadScript";
+
+const ZENDESK_SCRIPT_SRC =
+  "https://static.zdassets.com/ekr/snippet.js?key=57ad1fbc-d174-4561-8102-b3c8d98436d5";
```

Comment:
nit: Should this be in here or `settings.ts` ?

Reply:
Personally, I would have implemented it as an environment variable that is then mapped through, rather than as a hardcoded static. That method aligns more to how we handle existing things like this, and makes the code more flexible when others are deploying their own versions of it.

This would be mapped in https://github.com/sparkletown/sparkle/blob/staging/src/secrets.js (which isn't necessarily the best filename, since most of these values aren't actually secret.. particularly when mapped to the frontend.. might be worth renaming it `env` or similar? Realistically.. this file is actually what most of `settings.ts` would end up as one day I think.. so maybe it should be merged there or similar?)

Reply:
Backlogged these improvements in https://github.com/sparkletown/internal-sparkle-issues/issues/100